docker/nginx/site.conf

```diff
@@ -11,12 +12,12 @@ server {
         try_files $uri =404;
         fastcgi_split_path_info ^(.+\.php)(/.+)$;
         fastcgi_pass php-fpm:9000;
-        fastcgi_read_timeout 90;
+        fastcgi_read_timeout 3000;
```

Comment:
Why would you do that?

Reply:
I made the timeout longer for Docker Desktop. To prevent `504 Gateway Timeout`.
(https://github.com/xXSchrandXx/wsc-dockerized/blob/main/docker/nginx/site.conf)
Forgot to remove this for this request...

Reply:
The timeout is only important during installation, since Docker Desktop with WSL processes all the data more slowly. (Even slower if Intelliphense or other programs pause the container for every change. But I didn't understand that exactly.)